src/js/express.js

```diff
@@ -1,44 +1,67 @@
-// An [Express.js](http://expressjs.com/) instance represented as a Fluid component.
-//
-// By default, this instance does nothing useful.  Your instance will need to add at least one `gpii.express.router`
-// or `gpii.express.middleware` child component that will respond to incoming requests.  See those grades for more
-// details.
-//
-// Express configuration options are set using `options.config.express`.  The `options.config.express.views` variable
-// has special meaning for template renderers like `gpii-handlebars`.  It represents one or more directories containing
-// template subdirectories (`layouts`, `pages`, and `partials`). You may either have a single string value, or an array
-// of values.
-//
-// Each value will be passed through `fluid.model.resolvePath`, so that references to Fluid component packages can
-// be resolved relative to where they are installed, regardless of whether they are immediate or inherited dependencies.
-// You should be using these references, which generally are of the form `%npm-package-name/path/within/package`.
-//
-// See the `gpii-handlebars` package for more details about view directories.
-//
+/*
+
+    An [Express.js](http://expressjs.com/) instance represented as a Fluid component.  See the documentation for details:
+
+    https://github.com/GPII/gpii-express/blob/master/docs/express.md
+
+*/
 "use strict";
 var fluid = require("infusion");
 var gpii  = fluid.registerNamespace("gpii");
-require("./configholder");
 
 var express  = require("express");
 
 fluid.registerNamespace("gpii.express");
 
-// Look up the path (parent, sibling, child) relationships for a node.
-// Used in constructing the hierarchy we will use to wire together everything in our onCreate method.
-//
-// This does not require an invoker as it is basically a static method.
+/**
+ *
+ * @param mapOrArray {Object} - A map or array to be sorted.
+ * @returns {Array} - An array of elements, ordered by namespace and priority.
+ *
+ * Sorts an array or map by `priority`: http://docs.fluidproject.org/infusion/development/Priorities.html
+ *
+ * As maps do not preserve order, an array is always returned.  When used with a map, the keys will be discarded.
+ *
+ */
+gpii.express.orderByPriority = function (mapOrArray) {
```

Comment:
I tried this, but needed a newer version of Infusion to pick this up.  I updated to `2.0.0-dev.20160309T145652Z.9f515ec`, but got very odd errors.  When my debugging breakpoints in the infusion dependency weren't being hit, I figured out that the infusion from `kettle` was likely being used.  It seems to me like I need a newer matching set of `kettle` and `node-jqunit` to go with whatever `infusion` I use.  I looked briefly at the latest version of your KETTLE-32 branch, there is a different implementation of the priority ordering stack in the version used there.

Please help me identify a "matching set" for these three dependencies.